pkg/operator/ceohelpers/unsupported_override.go

```diff
@@ -56,3 +56,44 @@ func isUnsupportedUnsafeEtcd(spec *operatorv1.StaticPodOperatorSpec) (bool, erro
 		return false, nil
 	}
 }
+
+// IsUnsupportedEtcdStorage returns true if
+// useUnsupportedNonProductionUnstableStorageEtcd key is set
+// to any parsable value
+func IsUnsupportedEtcdStorage(spec *operatorv1.StaticPodOperatorSpec) (bool, error) {
```

Comment:
(No action required) Maybe refactor `isUnsupportedUnsafeEtcd` for reuse since the only difference is the name of the key?

Reply:
yeah for sure I just hacked up the poc for review by another team will make this cleaner